File: python coding challanges/01-basic-challenges-1/09-count-vowels/count-vowels.py
def count_vowels(string):
    lower_case_string = string.lower()
    
    count = 0
    
    vowels = {'a', 'e', 'i', 'o', 'u'}
    
    for char in lower_case_string:
        if char in vowels:
            count += 1
            
    return count

input_string = "Hello World"
vowel_count = count_vowels(input_string)
print(vowel_count)  # Output: 3

File: javascript coding challenges/01-basic-challenges-1/09-count-vowels/count-vowels.js
// function countVowels(string) {
//   const lowerCaseString = string.toLowerCase();
//   let count = 0;
//   for (let i = 0; i < lowerCaseString.length; i++) {
//     const vowels = ["a", "e", "i", "o", "u"];
//     if (vowels.includes(lowerCaseString[i])) count = count + 1;
//   }

//   return count;
// }

function countVowels(string) {
  const toLowerCase = string.toLowerCase(); 
  let count = 0;

  for (let i = 0; i < toLowerCase.length; i++) {
    if (
      toLowerCase[i] === "a" ||
      toLowerCase[i] === "e" ||
      toLowerCase[i] === "i" ||
      toLowerCase[i] === "o" ||
      toLowerCase[i] === "u"
    ) {
      count += 1; 
    }
  }
  return count;
}

module.exports = countVowels;



